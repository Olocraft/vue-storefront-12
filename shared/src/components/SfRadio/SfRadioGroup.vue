<template>
  <fieldset
    aria-required="true"
    aria-invalid="true"
    aria-describedby="errorMessage"
    role="radiogroup"
    class="flex flex-col"
  >
    <SfRadio
      name="radio-group-1"
      :invalid="true"
      value="one"
    />
    <SfRadio
      name="radio-group-2"
      value="two"
    />
    <span
      id="errorMessage"
      class="block col-span-2 mt-1 text-sm font-medium text-negative-600 font-body"
    >Error message</span>
  </fieldset>
</template>

<script>
import SfRadio from './SfRadio.vue'
export default {
  name: 'SfRadioGroup',
  components: {
    SfRadio
  }
}
</script>
