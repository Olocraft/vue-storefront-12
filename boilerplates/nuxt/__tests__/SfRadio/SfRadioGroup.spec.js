import { shallowMount } from '@vue/test-utils'
import SfRadioGroup from '../../components/SfRadio/SfRadioGroup.vue'

let wrapper

describe('SfRadioDense.vue', () => {
  const name = 'radio-group'
  beforeEach(() => {
    wrapper = shallowMount(SfRadioGroup, {
      propsData: {
        name
      }
    })
  })
  it('render component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
