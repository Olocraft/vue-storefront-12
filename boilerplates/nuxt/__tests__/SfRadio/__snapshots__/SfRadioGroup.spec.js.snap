// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SfRadioDense.vue render component correctly 1`] = `
<fieldset aria-required="true" aria-invalid="true" aria-describedby="errorMessage" role="radiogroup" class="flex flex-col" name="radio-group">
  <sfradio-stub name="radio-group-1" value="" invalid="true"></sfradio-stub>
  <sfradio-stub name="radio-group-2" value=""></sfradio-stub> <span id="errorMessage" class="block col-span-2 mt-1 text-sm font-medium text-negative-600 font-body">Error message</span>
</fieldset>
`;
